docs/how-to-guides/running-autoware-without-cuda.md

```diff
@@ -0,0 +1,34 @@
+# Running Autoware without CUDA
+
+Two following algorithms in Autoware Universe depend on a CUDA environment.
+
+- 2D/3D object detection
+- traffic light detection
+
+However, it is still possible to run Autoware in the environment without CUDA.
+The following subsections briefly explain how to run these algorithms in such an environment.
+
+!!! warning
+
+    It is highly recommended to prepare a CUDA environment to achieve better performance.
+
+## Running 2D/3D object detection without CUDA
+
+As for the object detection system in Autoware, the following three packages require the CUDA environment:
+
+- `lidar_centerpoint`
+- `lidar_apollo_instance_segmentation`
+- `tensorrt_yolo`
+
+Instead of these modules, you can use the `euclidean_cluster` module, which is a 3D object detection algorithm that does not require a CUDA environment (refer to [the readme](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/euclidean_cluster) for detail).
+
+## Running traffic light detection without CUDA
+
+As for the traffic light recognition system in Autoware, there are two package that require the CUDA environment:
```

Comment:
Rewording suggestion.

```suggestion
For traffic light recognition (both detection and classification), there are two modules that require CUDA:
```

Reply:
Thank you for the suggestion!
I have reflected the comments at here https://github.com/autowarefoundation/autoware-documentation/pull/128/commits/2f9ea059376f62b1fc11ba14173261e74aac19fb